pkgs/development/ocaml-modules/mirage-crypto/default.nix

```
@@ -1,23 +1,21 @@
-{ lib, fetchurl, buildDunePackage, ounit, cstruct, dune-configurator, eqaf, pkg-config
+{ lib, fetchurl, buildDunePackage, ounit2, cstruct, dune-configurator, eqaf, pkg-config
 , withFreestanding ? false
 , ocaml-freestanding
 }:
 
 buildDunePackage rec {
-  minimumOCamlVersion = "4.08";
+  minimalOCamlVersion = "4.08";
 
   pname = "mirage-crypto";
-  version = "0.10.5";
+  version = "0.10.6";
```

```suggestion
  version = "0.10.7";
```
`0.10.7` is available. Is `0.10.6` still desirable?

---

Added an update.